app/actions/v3/service_instance_delete.rb

```diff
@@ -55,10 +55,18 @@ def delete
       end
 
       def poll
-        result = client.fetch_service_instance_last_operation(
-          service_instance,
-          user_guid: service_event_repository.user_audit_info.user_guid
-        )
+        logger = Steno.logger('cc.action.service_instance_delete')
+
+        begin
+          result = client.fetch_service_instance_last_operation(
+            service_instance,
+            user_guid: service_event_repository.user_audit_info.user_guid
+          )
+        rescue HttpRequestError, HttpResponseError, Sequel::Error => e
```

Comment:
Before this change a service instance delete operation continued polling when an exception was thrown, because below `StandardErrors` are caught and then the polling continued (even though, the operation was set to failed confusingly).
After this change, when the poll request fails, the polling will continue and the operation will remain in state `in progress`.